kratos/elements/embedded_nodal_variable_calculation_element_simplex.h

```diff
@@ -0,0 +1,432 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+#if !defined(KRATOS_EMBEDDED_NODAL_VARIABLE_CALCULATION_ELEMENT_H_INCLUDED )
+#define  KRATOS_EMBEDDED_NODAL_VARIABLE_CALCULATION_ELEMENT_H_INCLUDED
+
+// System includes
+#include <string>
+#include <iostream>
+
+
+// External includes
+
+
+// Project includes
+#include "containers/array_1d.h"
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/serializer.h"
+#include "includes/variables.h"
+#include "geometries/geometry.h"
+#include "utilities/math_utils.h"
+#include "utilities/geometry_utilities.h"
+
+// Application includes
+
+
+namespace Kratos
+{
+
+///@addtogroup KratosCore
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// An element to compute the a nodal variable from an embedded skin
+/**
+ */
+template< class TVarType >
+class EmbeddedNodalVariableCalculationElementSimplex : public Element
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of EmbeddedNodalVariableCalculationElementSimplex
+    KRATOS_CLASS_POINTER_DEFINITION(EmbeddedNodalVariableCalculationElementSimplex);
+
+    /// Node type (default is: Node<3>)
+    typedef Node <3> NodeType;
+
+    /// Geometry type (using with given NodeType)
+    typedef Geometry<NodeType> GeometryType;
+
+    /// Definition of nodes container type, redefined from GeometryType
+    typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+
+    /// Vector type for local contributions to the linear system
+    typedef Vector VectorType;
+
+    /// Matrix type for local contributions to the linear system
+    typedef Matrix MatrixType;
+
+    typedef std::size_t IndexType;
+
+    typedef std::size_t SizeType;
+
+    typedef std::vector<std::size_t> EquationIdVectorType;
```

Comment:
```suggestion
    typedef Element::EquationIdVectorType EquationIdVectorType;
```
 better not manually redefine types
=> same apples also to your other typedefs

Reply:
True. Done. Thanks for reporting.